doc/source/architecture.rst

```diff
@@ -157,31 +156,8 @@ For more information, see :ref:`run-server`.
 All tools can be accessed from the *PMM Server* web interface (landing page).
 For more information, see :ref:`using`.
 
-.. _scenarios:
-
-Deployment Scenarios
---------------------
-
-PMM is designed to be scalable for various environments.
-Depending on the size and complexity of your infrastructure,
-you can deploy it in several ways.
-
-Simple Scenario
-***************
-
-If you have just one MySQL or MongoDB server,
-you can install and run both modules
-(*PMM Client* and *PMM Server*)
-on this one database host.
-
-Typical Scenario
-****************
-
-It is more typical to have several MySQL and MongoDB server instances
-distributed over different hosts.
-In this case, you can run *PMM Server* on a dedicated monitoring host,
-and install *PMM Client* on every database host that you want to monitor.
-Data from hosts will be aggregated on the PMM Server.
+.. DEPRECATED: moving deployment related information to the dedicated section
```

Comment:
"to a dedicated section" or "to the dedicated deployment section"

Reply:
"To the dedicated deployment section". It's a comment anyway which will not be rendered